### DISCUSSION TITLE
DEPLOYMENT 404 ERROR PLEASE HELP

### DISCUSSION THREAD

**Discussion by ailaktt** [community/community#132237] [Copilot Conversations]
### Select Topic Area

Question

### Body

HI I NEED HELP WITH MY DEPLOYMENT ASAP
Here is my repo link: https://github.com/ailaktt/WD-SEATWORK-4
While using VS Code, there is not a problem with any of my pages, but when it was deployed in github, the deployment link isn't working. But as I check the VS CODE live, the website is perfect. 
Please help me fix this!!

**Comment by Akash1134** [comment]
Hi @ailaktt ,

Please report issues related to VSCode in [our repo](https://github.com/microsoft/vscode/issues). 

It's possible that another GitHub user might have run into this same issue and can help, but the GitHub Community Discussions focuses primarily on topics related to GitHub itself or collaboration on project development and ideas. We want to make sure you’re getting the best support you can, but this space may not be the right place for this particular topic.

Thanks!
